src/app/(admin)/new_dashboard/guides/hooks/useSearchGuides.ts

```diff
@@ -22,11 +33,17 @@ export const useSearchGuides = (initialGuides: Guide[]) => {
     setSelectedTag(value || "All");
   };
 
+  const handleSortChange = (event: SelectChangeEvent<string> ) => {
+    setSortCriteria(event.target.value as "rating" | "date");
```

Comment:
you wrote the `"rating" | "date"` in this file many many times, please extract this to a const and replace every occurance of this hard coded option

https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

Reply:
import Type `SortCriteria `